docs/public-networks/reference/cli/options.md

```diff
@@ -5186,6 +5186,49 @@ The default is `2000`.
 For private networks, we recommend setting this value to the maximum number of transactions that fit
 in a block in your network.
 
+### `tx-pool-max-prioritized-by-type`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--tx-pool-max-prioritized-by-type=<TYPE=INTEGER>
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--tx-pool-max-prioritized-by-type=BLOB=6
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+BESU_TX_POOL_MAX_PRIORITIZED_BY_TYPE=BLOB=6
+```
+
+</TabItem>
+
+<TabItem value="Configuration file" label="Configuration file">
+
+```bash
+tx-pool-max-prioritized-by-type="BLOB=6"
+```
+
+</TabItem>
+
+</Tabs>
+
+The maximum number of transactions that can be held in the top-priority section of the transaction pool.
+Increasing the `BLOB` value increases the limit on prioritized [blob transactions](../../concepts/transactions/types#blob-transactions) in the transaction pool. 
+This allows blob transactions to be processed faster and prioritized over others in the queue based on factors such as type, fees, or importance.
```

Comment:
it's not really to prioritize blobs over other tx, it's more that keeping blobs around is expensive (they are big) so we only want to keep a small number (default 6, since max 6 in a block, but you can adjust as needed). 

https://github.com/hyperledger/besu/pull/6940

Reply:
maybe something like
Keeping this layer sorted is costly so it makes sense to limit this layer to transactions that could fit in a block. Since very few blob txs can be included in a block (currently max 6), tuning the max number of transactions by type also helps with keeping the prioritized layer performant.

Reply:
@macfarla. Is the new wording better/correct?